.github/workflows/python.yaml

```diff
@@ -96,6 +96,7 @@ jobs:
           - { os: self-hosted, module: gpu, python-version: 3.9}
           - { os: ubuntu-latest, module: jax, python-version: 3.9 }
           - { os: juicefs-ci, module: kubernetes-juicefs, python-version: 3.9 }
+          - { os: slurm-ci, module: slurm, python-version: 3.9 }
```

Comment:
os should be named to `ubuntu-latest`, otherwise github has no idea to allocate runner for you.